tfe/provider.go

```diff
@@ -214,7 +214,7 @@ func getClient(tfeHost, token string, insecure bool) (*tfe.Client, error) {
 
 	// If we still don't have a token at this point, we return an error.
 	if token == "" {
-		return nil, fmt.Errorf("required token could not be found")
+		return nil, fmt.Errorf("Required token could not be found. Please set the token in the provider configuration, or using the TFE_TOKEN environment variable.")
```

Comment:
I just realized we should probably emphasize best practice and not accidentally advise people to put their token as a string literal in the configuration 😅  How about this?

> Required token could not be found. Please set the token using an input variable in the provider configuration block by using the TFE_TOKEN environment variable.

Reply:
Makes sense 😄